tools/gcms2isocor/gcms2isocor.xml

```diff
@@ -0,0 +1,37 @@
+<tool id="gcms2isocor" name="Conversion GCMS PostRun Analysis to IsoCor format" version="0.1.0" profile="20.09">
```

Comment:
```suggestion
<tool id="gcms2isocor" name="Conversion GCMS PostRun Analysis to IsoCor format" version="0.2.1" profile="20.09">
```

Reply:
It would be even better to use a macro for the version to make sure the tool version is always the container version

Reply:
ok, thanks for this suggestion. 
I have modified the PR to take into account the macros @TOOL_VERSION@ and @VERSION_SUFFIX@.